zerver/webhooks/ansibletower/fixtures/ansibletower_system_job_successful.json

```diff
@@ -0,0 +1,11 @@
+{
+    "status": "successful",
+    "name": "Cleanup Job Details",
+    "started": "2018-09-13T18:57:15.891742+00:00",
+    "traceback": "",
+    "friendly_name": "System Job",
+    "created_by": "admin",
+    "url": "http://awx.example.co.uk/#/jobs/system/2721",
+    "finished": "2018-09-13T19:00:39.919286+00:00",
+    "id": 2721
+}
```

Comment:
Probably should have pointed this out earlier, but I don't think you need to prefix fixture names with `ansibletower_`. This is largely unnecessary and we stopped doing that a long time ago if I remember correctly. If there is a mistake in our docs that suggests this sort of naming scheme, please let me know! :)

Reply:
Yeah, I don't think it's like that in the docs now. I think it was back when I made my first integration for Zabbix a while back, and then just got stuck in the habit of it. Those are all changed now.